src/overlays/actors/ovl_En_Dno/z_en_dno.c

```diff
@@ -36,94 +84,949 @@ const ActorInit En_Dno_InitVars = {
+s32 func_80A7361C(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnDno* this = THIS;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72598.s")
+    *dList = NULL;
+    if (limbIndex == 9) {
+        rot->x += this->unk_466;
+    }
+    return 0;
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A725E0.s")
+void func_80A73654(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx) {
+    static Vec3f D_80A73B40 = { 0.0f, 0.0f, 0.0f };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A725F8.s")
+    Gfx* gfxOpa;
+    Gfx* gfxXlu;
+    Vec3f sp84;
+    EnDno* this = THIS;
+    s32 pad;
+    s32 phi_v0 = false;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72AE4.s")
+    if (*dList != NULL) {
+        switch (this->unk_452) {
+            case 0:
+                if ((limbIndex != 25) && (limbIndex != 26) && (limbIndex != 13) && (limbIndex != 15) &&
+                    (limbIndex != 16) && (limbIndex != 14)) {
+                    phi_v0 = true;
+                }
+                break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72B3C.s")
+            case 1:
+                if ((limbIndex != 26) && (limbIndex != 13) && (limbIndex != 15) && (limbIndex != 16) &&
+                    (limbIndex != 14)) {
+                    phi_v0 = true;
+                }
+                break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72B84.s")
+            case 2:
+                if ((limbIndex != 15) && (limbIndex != 16)) {
+                    phi_v0 = true;
+                }
+                break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72BA4.s")
+            case 3:
+                if ((limbIndex != 16) && (limbIndex != 14)) {
+                    phi_v0 = true;
+                }
+                break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72C04.s")
+            case 4:
+                if ((limbIndex != 15) && (limbIndex != 14)) {
+                    phi_v0 = true;
+                }
+                break;
+        }
+    }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72CF8.s")
+    if (phi_v0 == true) {
+        OPEN_DISPS(globalCtx->state.gfxCtx);
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72D8C.s")
+        func_8012C28C(globalCtx->state.gfxCtx);
+        if (limbIndex == 13) {
+            Matrix_Scale(this->unk_454, this->unk_454, this->unk_454, 1);
+            SysMatrix_InsertXRotation_s(this->unk_45C, 1);
+        }
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Dno/func_80A72FAC.s")
+        gfxOpa = POLY_OPA_DISP;
+        gSPMatrix(gfxOpa, Matrix_NewMtx(globalCtx->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        gSPDisplayList(gfxOpa + 1, *dList);
```

Comment:
```suggestion
        gSPDisplayList(&gfxOpa[1], *dList);
```
I mentioned this in another review, if this matches we should consider which style we prefer.

Reply:
My preference is `&gfxOpa[1]`